emperor/support_files/js/view.js

```diff
@@ -79,8 +81,22 @@ function DecompositionView(decomp) {
    */
   this.lines = {'left': null, 'right': null};
 
+  /**
+   * Whether or not the view relies on a PointCloud to display the data.
+   * @type {Bool}
+   */
+  Object.defineProperty(this, 'usesPointCloud', {
+    value: (this.decomp.length > 20000) || asPointCloud,
```

Comment:
it might be nice to be able to override this threshold?

Reply:
It can be overridden with the `asPointCloud` boolean value. This value can be changed from the constructor of this class. I originally added this to make testing quicker, but in general can be useful for other purposes.

Reply:
Is a plot with 20K slow and terrible? Just wondering how was this number selected?